Makefile

```diff
@@ -8,6 +8,19 @@
 GOBIN = $(shell pwd)/build/bin
 GO ?= latest
 
+# Compare current go version to minimum required version. Exit with \
+# error message if current version is older than required version.
+ver = $(shell go version)
+ver2 = $(word 3, ,$(ver))
+cur_ver = $(subst go,,$(ver2))
+#Set min_ver to the mininum required Go version such as "1.12.1"
+min_ver := 1.12.1
```

Comment:
What about moving this to the top of the block so stands out a little more?

If we're going to insist on a minor version, should we use the latest, `1.12.5`? Otherwise, `1.12` is fine.
@treeder thoughts?

Reply:
I think any 1.12 will work ya? So might as well leave it as 1.12.1. 

Looks good to me though, let's try it (I'm no make expert either).

Reply:
`1.12.1` disallows `1.12`, so I think either `1.12` or `1.12.5` makes more sense.

Reply:
Oh, I see, no 1.12.0? They don't strictly follow semver with Go?

Reply:
No they don't. I don't remember why and I can't find any reference, but :man_shrugging: